Assets/Modules/Obstacles/Scripts/ObstacleEntity.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ObstacleEntity : MonoBehaviour
```

Comment:
This is a good start. The toggles are indeed Serializefields that we can switch on and off in the inspector

Reply:
Also I think something went wrong when you wanted to create a new branch for the obstacle entity implementation and it was pushed here. No problem at all. We can merge this branch and move forward with the new one :)

Reply:
Ah yes, because I was going to work on the Obstacle entity, but then decided to go back to the movement, and I wasn't really sure how to do that haha so I just changed branches and brought the changes to this branch. sorry!